src/endpoints/minidump.rs

```diff
@@ -135,7 +135,7 @@ fn process_minidump(
         .and_then(|collect_state| match collect_state {
             (Some(file), Some(sources)) => Ok(ProcessMinidump {
                 file,
-                sources,
+                sources: Arc::new(sources),
```

Comment:
For `/symbolicate` there's a fallback to the preconfigured sources, we should have this here as well if we want to go forward with this.